snapshots/windows/windows.go

```diff
@@ -185,7 +188,7 @@ func (s *snapshotter) Mounts(ctx context.Context, key string) ([]mount.Mount, er
 	return s.mounts(snapshot), nil
 }
 
-func (s *snapshotter) Commit(ctx context.Context, name, key string, opts ...snapshots.Opt) error {
+func (s *snapshotter) Commit(ctx context.Context, name, key string, opts ...snapshots.Opt) (err error) {
```

Comment:
`(err error)` nice!

Reply:
could you mind to update defer with `retErr`? aligned with codebase style :)

Reply:
Changing those three instances of `rerr` to `retErr` makes `retErr` now outnumbers `rerr`... I assume this is a somewhat-recent codebase style change.

Reply:
Sorry, I didn't make it clear. 

```go
func (s *snapshotter) Commit(ctx context.Context, name, key string, opts ...snapshots.Opt) (retErr error) {
      ...
      defer func() {
           if retErr != nil {
			if rerr := t.Rollback(); rerr != nil {
				log.G(ctx).WithError(rerr).Warn("failed to rollback transaction")
			}
		}
      }()
      ...
}
```

WDYT?

Reply:
Oh, I see. Yeah, that makes sense. I don't recall any bare `return` calls in this code, so this should have limited impact. Happy to make that change tonight.